apps/storybook/src/Table.stories.tsx

```diff
@@ -2801,3 +2801,255 @@ ColumnManager.args = {
     },
   ],
 };
+
+export const StickyColumns: Story<Partial<TableProps>> = (args) => {
```

Comment:
Need to have story with sticky, draggable and column manager to play around and be sure nothing is getting broken.

Reply:
Added second `Full` story with bunch of stuff